capdl-loader-app/include/capdl.h

```diff
@@ -473,6 +473,14 @@ static inline seL4_CapRights_t CDL_seL4_Cap_Rights(CDL_Cap *cap)
                               !!(cap->rights & CDL_CanWrite));
 }
 
+static inline uint32_t CONST seL4_CapRights_get_capAllowAllRights(seL4_CapRights_t seL4_CapRights)
```

Comment:
This return value should be `bool`?

Reply:
Oh right, I was meaning to come back to this function. I based it directly on the functions automatically produced by the bitfield generator, and wasn't sure about the name, types, or even whether this was the right location for it.

I'm happy to change it to return `bool`, but should it have a simpler name and does it make sense for it to be here?